doc/source/ray-core/examples/automl_for_time_series.ipynb

```diff
@@ -0,0 +1,592 @@
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ray.init(ignore_reinit_error=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will break up our logic into several functions and a Ray [task](ray-remote-functions).\n",
+    "\n",
+    "The Ray task is `train_and_evaluate_fold`, which contains all the logic necessary to fit and evaluate a model on a CV fold of data. We structure our task to take in a dataset and indices splitting it into train and test - that way, we can keep one instance of the dataset in the Ray object store and split it in each task separately. We are defining this as a Ray task as we want all folds to be evaluated in parallel on a Ray cluster - Ray will handle all orchestration and execution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@ray.remote\n",
+    "def train_and_evaluate_fold(\n",
+    "    model: _TS,\n",
+    "    df: pd.DataFrame,\n",
+    "    train_indices: np.ndarray,\n",
+    "    test_indices: np.ndarray,\n",
+    "    label_column: str,\n",
+    "    metrics: Dict[str, Callable[[pd.Series, pd.Series], float]],\n",
+    "    freq: str = \"D\",\n",
+    ") -> Dict[str, float]:\n",
+    "    try:\n",
+    "        statsforecast = StatsForecast(\n",
+    "            df=df.iloc[train_indices], models=[model], freq=freq\n",
+    "        )\n",
+    "        forecast = statsforecast.forecast(len(test_indices))\n",
+    "        return {\n",
+    "            metric_name: metric(\n",
+    "                df.iloc[test_indices][label_column], forecast[model.__class__.__name__]\n",
+    "            )\n",
+    "            for metric_name, metric in metrics.items()\n",
+    "        }\n",
+    "    except Exception:\n",
+    "        return {metric_name: None for metric_name, metric in metrics.items()}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`evaluate_models_with_cv` is a driver function to run our optimization loop. We take in a list of models (with their parameters already set) and the dataframe.\n",
+    "\n",
+    "The dataframe is put into the Ray object store and reused, which means we only need to serialize it once. That way, we avoid an {doc}`anti-pattern </ray-core/patterns/pass-large-arg-by-value>`.\n",
```

Comment:
could we just do "{doc}`/ray-core/patterns/pass-large-arg-by-value`" so the title of the anti-pattern is included.